index.js

```diff
@@ -47,7 +56,9 @@ function jaegerPlugin(fastify, opts, next) {
     fastify.decorateRequest('jaeger', api);
   }
 
-  if (!opts.disable) {
+  const jaegerDisabled = 'disable' in tracerConfig ? tracerConfig.disable : getBooleanEnv('JAEGER_DISABLED', true);
+
+  if (!jaegerDisabled) {
```

Comment:
This is just an idea that came to me now - wouldn't it be more correct if we checked for the presence of `registerInjector` and `registerExtractor` functions, like we do with `close`?

Reply:
sure, good idea